src/lib/database/migrations/1616201639503-V42_AddAllowedRolesIntoGiveaways.ts

```diff
@@ -0,0 +1,20 @@
+import { MigrationInterface, QueryRunner, TableColumn } from 'typeorm';
+
+export class V42AddAllowedRolesIntoGiveaways1616201639503 implements MigrationInterface {
+	public async up(queryRunner: QueryRunner): Promise<void> {
+		await queryRunner.addColumn(
+			'giveaway',
+			new TableColumn({
+				name: 'allowed_roles',
```

Comment:
why is this snake cased and not kebab cased?

Reply:
PostgreSQL's naming convention is to use `snake_case`.

The reason why I didn't for the guilds table, is because users find `kebab-case` easier, either way, it doesn't follow conventions per-se, as dots are precisely pretty against the PostgreSQL's standards (and more so that it has specific a meaning).

*It definitely didn't occur to me 2 hours later that I could have replaced `_` with `-` in the maps when generating the settings trees.*